compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -1187,42 +1148,11 @@ public void visit(BLangRecordTypeNode recordTypeNode) {
 
         for (BLangSimpleVariable bLangSimpleVariable : recordTypeNode.fields) {
             bLangSimpleVariable.typeNode = rewrite(bLangSimpleVariable.typeNode, env);
+//            bLangSimpleVariable.expr = rewrite(bLangSimpleVariable.expr, env);
```

Comment:
Commented code? Is this visited elsewhere?

Reply:
Yes. when visiting the generated closures(lambda functions).

Reply:
Removed the comment